interface/wx/dynarray.h

```diff
@@ -271,6 +271,14 @@ class wxArray<T>
     */
     wxObjArray(const wxObjArray& array);
 
+    /**
+        Constructs the container with the contents of the initializer_list @list.
+
+        @since 3.2.2
+    */
+    template<typename T>
+    wxArray(std::initializer_list<T> list);
```

Comment:
Do we need to add documentation for the other constructors as well? `wxObjArray` and `wxSortedArray`?

Reply:
To be honest, I have no idea what's going on with the documentation of this class as it seems to document several different classes inside a single class declaration... But, to answer the question, I don't think we need to continue doing it like this, just this should be enough, thanks.